server/executor/runner.go

```diff
@@ -81,6 +85,12 @@ func (r persistentRunner) handleDBError(run model.Run, err error) {
 	}
 }
 
+func (r persistentRunner) handleError(run model.Run, err error) {
+	if err != nil {
+		fmt.Printf("test %s run #%d trigger DB error: %s\n", run.TestID, run.ID, err.Error())
```

Comment:
Philosophical question: should we put this error handling on `EventEmitter` and make `Emit` return nothing?

Reply:
I prefer to handle it on the runner level. Mainly because this is a temporary thing, we should use a real logger here instead.